specs/PushNotifications/ToastNotifications-spec.md

```diff
@@ -0,0 +1,509 @@
+                    valueStringOverride = "{progressValueString}"
+                    status = "{progressStatus}" />
+            </binding>
+        </visual>
+    </toast>)";
+
+    winrt::AppNotification notification(payload);
+    notification.Tag(tag);
+    notification.Group(group);
+
+    // Assign initial values for first notification progress UI
+    winrt::AppNotificationProgressData data(1);
+    data.Title(L"Weekly playlist"); // Binds to {progressTitle} in xml payload
+    data.Value(0.6); // Binds to {progressValue} in xml payload
+    data.ValueStringOverride(L"15/26 songs"); // Binds to {progressValueString} in xml payload
+    data.Status(L"Downloading..."); // Binds to {progressStatus} in xml payload
+
+    notification.Progress(data);
+    winrt::AppNotificationManager::Default().Show(notification);
+}
+
+// Send subsequent progress updates
+winrt::Windows::Foundation::IAsyncAction UpdateProgressAsync()
+{
+    winrt::hstring tag = L"weekly-playlist";
+    winrt::hstring group = L"downloads";
+
+    // Assign new values
+    winrt::AppNotificationProgressData data(2 /* Sequence number */ );
+    data.Title(L"Weekly playlist"); // Binds to {progressTitle} in xml payload
+    data.Value(0.7); // Binds to {progressValue} in xml payload
+    data.ValueStringOverride(L"18/26 songs"); // Binds to {progressValueString} in xml payload
+    data.Status(L"Downloading..."); // Binds to {progressStatus} in xml payload
+
+    auto result = co_await winrt::AppNotificationManager::Default().UpdateAsync(data, tag, group);
+    if (result == winrt::AppNotificationProgressResult::AppNotificationNotFound)
+    {
+        LOG_HR_MSG(E_NOT_SET, "Toast Progress Update Failed since the previous notification update is missing or has been cleared by the user!");
+    }
+}
+```
+
+# Remarks
+
+## Registration
+
+The developer should always call the Notification Registration API first to register the current
+process as the target for Notification invokes. When the app is terminating, the developer should
+call Unregister() to free up COM and allow for subsequent invokes to launch a new process.
+
+## Event Handlers
+
+The developer should subscribe to Notification event handlers to handle invocations if the process
+happens to be already running. If this isn't done, the OS will launch a new process for every
+subsequent invocation.
+
+## Manifest Registration
+
+For MSIX, the COM activator GUID and the exe need to be specified in the manifest as part of the
+desktop and com extensions. In the example below, we use an activatorId =
+"1920DBA9-0F64-4F0D-8A4B-5E207B812E61". The launch args would need to be pre-set to
+"----AppNotificationActivated:" to ensure that WinAppSDK can process the payload as an
+AppNotification Kind.
+
+```xml
+<Extensions>
+    <desktop:Extension Category="windows.toastNotificationActivation">
+		<desktop:ToastNotificationActivation ToastActivatorCLSID="1920DBA9-0F64-4F0D-8A4B-5E207B812E61" />
```

Comment:
the indent is wrong here

Reply:
Ran .prettier and it looks like it resolved this issue.